hack/.notableofcontents

```diff
@@ -1,5 +1,6 @@
 docs/aks-installation.md
 docs/api.md
+docs/antctl.md
```

Comment:
This doc has table of contents and we need CI to ensure it's accurate, why adding it to the exception list?

Reply:
removed it.